caesars-palace/blocks/slider-card/slider-card.js

```diff
@@ -1,42 +1,75 @@
 export default function decorate(block) {
```

Comment:
WDYT of just calling this block `slider`? I wondering if that could potnetially allow for sliding different types of content. By default each slide item would be rendered as a card but variations could then be added to `slider` to support different slide designs.

Reply:
I think we should take it easy and start with card-slider. The behaviors of sliding elements may vary with the transitions or effects. We could always refactor if/when it is needed.

Reply:
That is why I was thinking `slider` could work att he start. It is simple for authors to understand. If/when a new design comes along that doesn't fit into existing slider pattern a decision can then be made to either extend `slider` or create a compeltely new block. Already I know the rewards section also needs to be a slider so I am hoping the same block could be used for that design as well.

Reply:
Well, the rewards section is a card slider? Not just a slider. There are cards in there? And I'd see it as variation of a card-slider, not an abstract slider component.